client/state/analytics/actions.js

```diff
@@ -18,13 +24,16 @@ const mergedMetaData = ( a, b ) => [
 
 const joinAnalytics = ( analytics, action ) =>
 	isFunction( action )
-		? dispatch => { dispatch( analytics ); dispatch( action ); }
+		? dispatch => {
+			dispatch( analytics );
+			dispatch( action );
+		}
```

Comment:
Ah yup, I don't like it either, but it violates our ESLint's `brace-style` (http://eslint.org/docs/rules/brace-style).